coalib/coala_main.py

```diff
@@ -180,7 +180,12 @@ def run_coala(console_printer=None,
             if not autoapply:
                 section['default_actions'] = ''
             elif force_show_patch:
-                section['default_actions'] = '*: ShowPatchAction'
+                # Append ShowPatchAction to default actions
+                try:
+                    section['default_actions'] = '*: ShowPatchAction, {}' \
```

Comment:
Explicit Line Continuation is making you CI red.
AppVeyor https://ci.appveyor.com/project/coala/coala/builds/22244867/job/q0baiyyplpipelte#L1659
Travis https://travis-ci.org/coala/coala/jobs/490937804#L1979